### DISCUSSION TITLE
Screen saver problem

### DISCUSSION THREAD

**Discussion by Lunatic-Spirit** [rocksdanister/lively#2516] [Q&A]
Hello, 
I have a small screensaver problem that I can't solve. I installed as said in the tutorial but the screensaver and wallpapers without the application too. I copied the files as said into the Windows folder, the update also works on the software but it is impossible to make the image appear on the screen saver settings. I'm posting below because pictures are worth a thousand words.
![drzr](https://github.com/user-attachments/assets/149af4cb-bfdb-4e06-b12d-99ce32c1d68b)
![test](https://github.com/user-attachments/assets/cb235bd7-dabf-407f-8365-52aecdd46dd0)
![er](https://github.com/user-attachments/assets/0fbdbad9-826f-4648-bb54-45b78a7ae036)

Thanks!

**Comment by rocksdanister** [comment]
That's normal, as long as the image appears on lively settings it's fine.

**Reply by Lunatic-Spirit** [reply]
The problem is that the wallpapers work but when I put the PC to sleep I end up with my basic fixed image :/

**Reply by rocksdanister** [reply]
What is fixed image? I don't follow.

If you mean wallpaper then this sounds like a different issue than the screensaver problem mentioned here.

**Reply by Lunatic-Spirit** [reply]
Ok I understood, this one is the screen saver not the PC startup screen which I got confused -_- sorry for being a stupid idiot thanks anyway :)
